spec/models/fingerprint_spec.rb

```diff
@@ -62,4 +62,56 @@
       expect(Fingerprint.with_deleted).to eq [fingerprint]
     end
   end
+
+  context 'using molfile' do
+    let(:polymer_molfile) {
+      <<-MOLFILE
+
+  Ketcher 08221616432D 1   1.00000     0.00000     0
+
+  2  1  0     0  0            999 V2000
+    2.8250   -5.1500    0.0000 R#  0  0  0  0  0  0  0  0  0  0  0  0
+    3.8250   -5.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0     0  0
+M  RGP  1   1   1
+M  END
+> <PolymersList>
+0
+0
+$$$$
+
+MOLFILE
+    }
+
+    it 'should standardized molfile a fingerprint' do
+      standard = Fingerprint.standardized_molfile(polymer_molfile)
+
+      expect(standard.include? 'R#').to be(false)
+      expect(standard.include? 'PolymersList').to be(false)
+    end
+
+    it 'should create a fingerprint' do
+      fp_id = Fingerprint.find_or_create_by_molfile(polymer_molfile)
+      fp = Fingerprint.find(fp_id)
+
+      expect(fp.fp0).to eq("0000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
It's more simple for me when I manually generated a 64-bits fingerprint. It's also easy to clearly see the fingerprint I want to test, that the generated fingerprint should be exactly like this. Can I just keep it like this?